### PR TITLE
Rename Kubeflow Notebooks meeting title to Kubeflow 2.0 and remove Kubeflow Notebooks APAC meeting.

### PR DESCRIPTION
**Pull request by ederign**
Please wait for @thesuperzapper review.

### CODE REVIEW

File: calendar/calendar.yaml [hunk 1]
```diff
@@ -98,7 +98,7 @@
   organizer: juliusvonkohout
 
 - id: kf037
-  name: Kubeflow Notebooks WG Meeting (US + EMEA)
+  name: Kubeflow Notebooks 2.0 WG Meeting (US + EMEA)
```
Comment (thesuperzapper) [inline calendar/calendar.yaml:101]: @ederign I think it's more clear if we add `+` in the middle, so it shows we are doing both the Notebooks 2.0 and WG meeting. 

```suggestion
  name: Kubeflow Notebooks 2.0 + WG Meeting (US + EMEA)
```

**Comment by thesuperzapper** [comment]
Thanks @ederign

/lgtm
/approve

**Comment by thesuperzapper** [comment]
@Bobgy or @jbottum can you approve this, as we don't have the 4pm meeting anymore, and it needs to be removed from the calender.

File: calendar/calendar.yaml [hunk 2]
```diff
@@ -510,4 +490,25 @@
     - |
       Notes & Agenda: http://bit.ly/kfp-meeting-notes
       Join at: https://meet.google.com/jkr-dupp-wwm
-  organizer: chensun
\ No newline at end of file
+  organizer: chensun
+
+- id: kf040
```
Comment (terrytangyuan) [inline calendar/calendar.yaml:495]: The meeting is still here?

Comment (terrytangyuan) [reply calendar/calendar.yaml:495]: Oh there’s an “until” field

**Review by terrytangyuan — APPROVED** [review]
`
Thanks!

/lgtm
/approve

**Pull request merged**